tests/integration/basics.rs

```diff
@@ -757,3 +757,26 @@ fn it_doesnt_warn_with_neither_config_nor_ignore() {
         .stdout(predicates::str::contains("neither").count(0).from_utf8())
         .stdout(predicates::str::contains("Done!").from_utf8());
 }
+
+#[test]
+fn it_processes_dot_github_directory_files() {
+    let template = dir("template")
+        .file(".github/foo.txt", "{{project-name}}")
+        .init_git()
+        .build();
+    let dir = dir("main").build();
+
+    Command::main_binary()
+        .unwrap()
+        .arg("gen")
+        .arg("--git")
+        .arg(template.path())
+        .arg("-n")
+        .arg("foobar-project")
+        .current_dir(&dir.path())
+        .assert()
+        .success()
+        .stdout(predicates::str::contains("Done!").from_utf8());
+
+    assert_eq!(dir.read("foobar-project/.github/foo.txt"), "foobar-project");
```

Comment:
I should note that I wrote a failing test first. this previously evaluated to '{{project-name}}' because `.github` contains the string `.git`